scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/ScalametaTokenizer.scala

```diff
@@ -194,16 +193,8 @@ class ScalametaTokenizer(input: Input, dialect: Dialect) {
       def nextToken() = legacyIndex += 1
       if (legacyIndex >= legacyTokens.length) return legacyIndex
 
-      if (dialect.allowSignificantIndentation && curr.token == COLON && aheadIsNewLine(
-          legacyIndex
-        )) {
-        curr.token = COLONEOL
-        emitToken()
-        legacyIndex += 2
```

Comment:
here we jumped +2 and consumed ':' and LF, but because we no longer do that, LF will be emitted correctly(related to comment above)